src/aspire/reconstruction/estimator.py

```diff
@@ -87,10 +86,9 @@ def src_backward(self):
             im = self.src.images(i, self.batch_size)
             batch_mean_b = self.src.im_backward(im, i) / self.src.n
             mean_b += batch_mean_b.astype(self.dtype)
-
         res = self.basis.evaluate_t(mean_b)
         logger.info(f"Determined adjoint mappings. Shape = {res.shape}")
-        return res
+        return np.squeeze(res)
```

Comment:
Are we reporting one shape for `res` and returning another? Is `squeeze` removing the `0` axis here as well? If so, maybe we should be explicit here too.